micr_XPCTzarr/dataset_description.json

```diff
@@ -0,0 +1,14 @@
+{
+	"Name": "micr_XPCTzarr",
+	"Authors": [
+		"Matthieu Chourrout",
+		"David Stansby",
+		"Claire L. Walsh",
+		"Peter D. Lee"
+	],
+	"BIDSVersion": "1.10.0",
+	"License": "CC-BY-4.0",
+	"DatasetDOI": "doi:10.15151/ESRF-DC-572252655",
+	"DatasetType": "raw",
+	"HowToAcknowledge": "Please cite this dataset as: Tafforeau, P., Walsh, C., Wagner, W. L.,  Daniyal J. Jafree, Bellier, A., Werlein, C., Kühnel, M. P., Boller, E., Walker-Samuel, S., Robertus, J. L., Long, D. A., Jacob, J., Marussi, S.,  Eeline Brown, Holroyd, N., Jonigk, D. D., Ackermann, M., & Lee, P. D. (2021). Complete brain from the body donor LADAF-2020-31 (Version 1) [dataset]. European Synchrotron Radiation Facility. https://doi.org/10.15151/ESRF-DC-572252655"
```

Comment:
@dstansby @Guillaume-Gaisne 
@ucbpclw mentioned that folowing your last metadata meeting, the way to acknowledge may change: can you please tag me once it does?

Reply:
Following our metadata meeting, it is the way to acknowledge the HOA portal that will change. We will use the article about the data portal that is currently under writing to cite the portal and then, for each dataset, users will have to cite the DOI of the dataset

Reply:
Thanks Guillaume, this is a TODO (@chourroutm) then! I'll update it ASAP.

Reply:
Again, since this is an example it doesn't have to be perfect. So I think what's here is fine now.